testdata/redundant-import-alias.go

```diff
@@ -1,12 +1,16 @@
 package fixtures
 
-import(
+import (
 	"crypto/md5"
-		"strings"
-		_ "crypto/md5"
```

Comment:
Can we please have this back in the test case?

Reply:
old testdata restored